Spanner/src/Session/CacheSessionPool.php

```diff
@@ -104,6 +104,17 @@
  *     ]
  * ]);
  * ```
+ *
+ * ```
+ * Database role configured on the pool will be applied to each session created by the pool.
+ * use Google\Cloud\Spanner\Session\CacheSessionPool;
```

Comment:
```suggestion

 * Database role configured on the pool will be applied to each session created by the pool.
 * ```
 * use Google\Cloud\Spanner\Session\CacheSessionPool;
```

Reply:
Done